quiz.css

```diff
@@ -0,0 +1,78 @@
+
+.flex {
+	display: flex;
+}
+
+img {
+	display: block;
+	margin: 0 auto;
+	margin-top: 20px;
+}
+.pdiscription {
+	text-align: justify;
+	padding-left: 20px;
```

Comment:
it looks a little funky to have padding only on the left.  what about setting padding to 10px on both left and right?  then the text will be centered a little better